recipes/tokyocabinet/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+
+./configure --prefix=$PREFIX \
+	    --with-zlib=$PREFIX \
+	    --with-bzip=$PREFIX
+
+make
+make check > /dev/null
```

Comment:
It's huge, and the build machine only shows the first 10000 lines or so. If something goes wrong we can't see whats written to stderr anymore.